src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -325,6 +320,17 @@ private void event(
     }
   }
 
+  private boolean getOptOutOfSessionFromOptions(@Nullable Map options) {
```

Comment:
This method can be written in a simpler without storing the result in a local variable and returning the value directly and using early returns to minimize nesting.

Reply:
Ok, made some updates.